test/org/traccar/TestIdentityManager.java

```diff
@@ -0,0 +1,122 @@
+package org.traccar;
+
+import org.traccar.database.*;
+import org.traccar.model.*;
+import java.util.concurrent.ConcurrentHashMap;
+
+import java.util.Map;
+
+public final class TestIdentityManager implements IdentityManager {
```

Comment:
This class seems to be overcomplicated. I prefer unit tests to be as simple as possible.

Reply:
Should i remove unused code and simplify the code, or you does not accept the whole approach?

Reply:
I think it's a bad idea for unit tests to have some internal state, but I like the idea of having a separate class for it. I guess to simplify the class to something close to the original one, possibly with some small additions/modifications.

Reply:
done

Reply:
You still internal state in this class.